index.d.ts

```diff
@@ -0,0 +1,214 @@
+/// <reference types="node" />
+/// <reference types="aws-lambda" />
+/// <reference types="connect" />
+
+import { IncomingMessage, ServerResponse } from 'http';
+import { Handler } from 'aws-lambda';
+import { ErrorHandleFunction } from 'connect';
+
+export = agent;
```

Comment:
Changed to using this syntax instead of `export default agent`.

But unfortunately this means that TypeScript users will need import the module differently from Babel users and users of the experimental ES Module support in Node.js core:

```js
// Style 1: Works only with TypeScript
import * as agent from 'elastic-apm-node'

// Style 2: Works only with Babel and `node --experimental-modules`
import agent from 'elastic-apm-node'
```

If a TypeScript user uses the second style, it will break.

Reply:
AFAIK, Typescript users can still use the second style by setting  allowSyntheticDefaultImports flag in compiler options.

Reply:
Oh you're right! I didn't know about that config option. Do you know how common that is to use and if it's well known?

Reply:
Yeah its pretty common, We have used it extensively in lots of projects where the code must run both on Node.js as well as on the browser.

Reply:
Cool I guess I'll just add some docs then describing how to use it just in case. If anything because we've already documented to Babel stuff, so it would be weird if we didn't also document this.

Reply:
After a bit of testing, it turns out that `allowSyntheticDefaultImports` only "silences" the errors, but the JavaScript code it produces still thinks that there should be a `default` property  on the export. But if I instead use `esModuleInterop`, it works as expected